tomcat-common/src/main/java/org/jboss/arquillian/container/tomcat/TomcatManager.java

```diff
@@ -100,6 +100,10 @@ public void undeploy(final String name) throws IOException, DeploymentException
         execute(command.toString(), null, null, -1);
     }
 
+    public void serverInfo() throws IOException {
+        execute(tomcatManagerCommandSpec.getServerInfoCommand(), null, null, -1);
+    }
+    
     public void list() throws IOException {
```

Comment:
It seems that by introducinb `serverInfo` `list` method became obsolete. Can you clean up unused code?